src/components/newsFeedV2/newsFeed.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styles from './newsFeed.css';
+import News from './news';
+import Box from '../boxV2';
+import logo from '../../assets/images/Lisk-Logo.svg';
+import ShowMore from '../showMore';
+
+class NewsFeed extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showMore: false,
+    };
+
+    props.getNewsFeed();
+  }
+
+  onShowMore() {
+    this.setState({ showMore: !this.state.showMore });
+  }
+
+  render() {
+    const onShowMore = this.state.showMore ? styles.showMore : '';
+
+    const filteredNewsFeed =
+      this.props.newsFeed.filter(feed => this.props.channels[feed.source]) || [];
+
+    return (
+      <Box className={`newsFeed-box ${styles.newsFeedBox}`}>
+        <header className={styles.header}>
```

Comment:
```suggestion
        <header>
```
`styles.header` is now undefined

Reply:
sorry Vit my mistakate... I remove this